automation/DataAggregator/S3Aggregator.py

```diff
@@ -257,6 +257,8 @@ def process_content(self, record):
     def drain_queue(self):
         """Process remaining records in queue and sync final files to S3"""
         super(S3Listener, self).drain_queue()
+        for crawl_id in self.browser_map.keys():
```

Comment:
Can we iterate on the values instead of the keys, and directly call `_create_batch` on these? Maybe there's a python idiom to do it all in one line?

Reply:
I updated to iterating on values. I think we could also do `map(self._create_batch, self.browser_map.values())`, but I think using a loop that names the values (`visit_id`) makes it more understandable as to what we're looping over.